### DISCUSSION TITLE
Only show battery percentage on systems that have a battery

### DISCUSSION THREAD

**Discussion by szczys** [catppuccin/tmux#200] [Show & Tell]
I share the same .tmux.conf across all systems. I finally added battery percentage display today, and I noticed that it is shown on a desktop system that doesn't have a battery (renders to 0%). I worked out this bit of code that will only display on systems that have a battery present. I'm on Linux Mint, your mileage may vary based on the distro you use and where it maps battery information:

![image](https://github.com/catppuccin/tmux/assets/367685/89efd0d2-8c63-40ed-a239-573eaacd4b3b)

```
# If system has a battery, show remaining percentage
if-shell '[ -d "/sys/class/power_supply/BAT0" ]' \
  'BATTERY="battery"' \
  'BATTERY=""'

set -g @catppuccin_status_modules_right "session host $BATTERY date_time"
```

**Comment by the-spanish-guy** [comment]
I was having the same problem, thanks
